src/openforms/registrations/api/views.py

```diff
@@ -86,13 +123,48 @@ def get_objects(self):
             catalogus["url"]: catalogus for catalogus in catalogus_data
         }
 
-        iotypen_data = client.get_all_informatieobjecttypen()
-        iotypen = [
-            {
-                "informatieobjecttype": factory(InformatieObjectType, iotype),
-                "catalogus": factory(Catalogus, catalogus_mapping[iotype["catalogus"]]),
-            }
-            for iotype in iotypen_data
-        ]
+        if "catalogus_domein" in filter_serializer.validated_data:
```

Comment:
we get back a `url` already when we retrieve a list of catalogi - can we not just use the catalogus URL directly as a filter parameter rather than having to pass the domein + rsin?

We'll need to store those values in the options, but for these filter endpoints, we can directly forward the URL query param, since these don't need to be stored, right?

Reply:
Hopefully this won't complicate things in the frontend